pubspec.yaml

```diff
@@ -5,10 +5,10 @@ author: Ian Luca de Godoy Serpa <ian.naph@gmail.com>
 homepage: https://github.com/ianldgs/material_search
 
 environment:
-  sdk: '>=1.8.0 <2.0.0'
+  sdk: '>=1.8.0 <=2.1.0'
```

Comment:
Maybe removing this limitation would be better, don't you think? What happens when dart releases 2.2?

Reply:
Mind following these instructions?
https://github.com/ianldgs/material_search/issues/30#issuecomment-418119069
If you can't, at my lunch time today I fix it...

Reply:
No rush, I will try to change it later today. Thanks!

Reply:
Great! Thank you very much for the PR.